packages/shared/src/components/filters/TagsFilter.spec.tsx

```diff
@@ -180,11 +197,14 @@ it('should follow a tag on click and remove filter alert if enabled', async () =
   const buttonDiv = await screen.findByTestId('tagCategoryTags');
   expect(buttonDiv).toBeVisible();
 
-  // eslint-disable-next-line testing-library/prefer-screen-queries
-  const button = await screen.findByText('#webdev');
-  button.click();
+  await act(async () => {
```

Comment:
Are you sure the act is needed here?

Reply:
This was just from the existing test, so I think it is safe to say, yes.

Reply:
You are right! Seems like we don't need it. Have removed it, including on the existing test.